src/main/java/teammates/ui/automated/PutCommentDocumentWorkerAction.java

```diff
@@ -0,0 +1,30 @@
+package teammates.ui.automated;
+
+import teammates.common.datatransfer.attributes.CommentAttributes;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+
+/**
+ * Task queue worker action: Puts searchable document for comment.
+ */
+public class PutCommentDocumentWorkerAction extends AutomatedAction {
+
+    @Override
+    protected String getActionDescription() {
+        return null;
+    }
+
+    @Override
+    protected String getActionMessage() {
+        return null;
+    }
+
+    @Override
+    public void execute() {
+        String commentId = getRequestParamValue(Const.ParamsNames.COMMENT_ID);
+        Assumption.assertNotNull(commentId);
+
+        CommentAttributes comment = logic.getComment(Long.valueOf(commentId));
```

Comment:
Erm, thx for the clarification on Slack and I will just repeat our discussed result here for others to read in case they are confused as well:

Comment Entity is created and put into DB during 
``` CommentAttributes createdComment = logic.createComment(comment);```

The task **PutCommentDocumentWorkerAction.java** is to put the Document of this comment into DB.

Sorry that I was confused about this part. Now it looks good to me